### PR TITLE
Add tests for calculateTime

### PR DESCRIPTION
**Pull request by kez-lab**
## Summary
- add a `commonTest` source set
- add unit tests verifying `calculateTime` for 12h and 24h cases
- document each test in `TimeCalculationTest`

## Testing
- `./gradlew :datetimepicker:desktopTest`

------
https://chatgpt.com/codex/tasks/task_e_68596c365cd88326a55c9cc0ca0ceda8

## Summary by CodeRabbit

- **Tests**
  - Added unit tests to verify time calculation functionality for various input formats, including 24-hour and 12-hour (AM/PM), as well as edge cases like midnight and noon.

### CODE REVIEW

File: datetimepicker/src/commonTest/kotlin/com/kez/picker/util/TimeCalculationTest.kt [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            year = currentYear,
+            monthNumber = currentMonth,
+            dayOfMonth = currentDate.dayOfMonth,
+            hour = 12,
+            minute = 30
+        )
+
+        assertEquals(expected, result)
+    }
+}
```
[the remark this answers is not in the document]
Comment (kez-lab) [reply datetimepicker/src/commonTest/kotlin/com/kez/picker/util/TimeCalculationTest.kt]: @coderabbitai 수정했습니다 [cdd6975](https://github.com/kez-lab/Compose-DateTimePicker/pull/9/commits/cdd69758eba9ac3dbbfa9a87b95b3b94adc1fb24)
